internal/strace/strace_test.go

```diff
@@ -89,26 +94,33 @@ func TestParseFileWritesToDifferentFiles(t *testing.T) {
 		},
 	}
 
-	secondFileInfo := strace.FileInfo{
+	secondFileWant := strace.FileInfo{
 		Path:  "pipe:[5]",
 		Write: true,
 		WriteInfo: strace.WriteInfo{
 			{
 				BytesWritten:  3684,
-				WriteBufferId: "7719f4b5f43eda5bf95d9cc31134c86711b5cbaa689441b570a487bf4b9c5e61",
+				WriteBufferId: "7b2c403bc9eee677758c2a575b2f8602b37f880f4fdb98ee98c30a74a5b9a52b",
 			},
 		},
 	}
-	want := []strace.FileInfo{firstFileInfo, secondFileInfo}
 
 	r := strings.NewReader(input)
 	res, err := strace.Parse(r)
 	if err != nil || res == nil {
 		t.Errorf(`Parse(r) = %v, %v, want _, nil`, res, err)
 	}
 	files := res.Files()
-	if !reflect.DeepEqual(files, want) {
-		t.Errorf(`Files() = %v, want  = %v`, files, want)
+	if len(files) != 2 {
+		t.Errorf(`Length of files = %v, want %v`, len(files), 2)
+	}
+	file1 := files[0]
+	file2 := files[1]
+	if file1.Path != firstFileWant.Path || file1.Write != firstFileWant.Write || !reflect.DeepEqual(file1.WriteInfo, firstFileWant.WriteInfo) {
```

Comment:
Testing relevant components separately since we can't test the actual paths stored in FileWriteBufferPaths as there is some randomness involved with CreateTemp()